js/src/Ice/Operation.js

```diff
@@ -244,7 +244,9 @@ function dispatchImpl(servant, op, request) {
     // Check to make sure the servant implements the operation.
     //
     const method = servant[op.servantMethod];
-    console.assert(typeof method === "function");
+    if (typeof method !== "function") {
+        throw new OperationNotExistException();
```

Comment:
This catches the case the application didn't implemented an operation, the other check we already have just catches operations missing from the Slice definition, for example client/server using different Slice definitions.